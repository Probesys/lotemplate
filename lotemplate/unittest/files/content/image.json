{"image": {"type": "image", "value": "lotemplate/unittest/files/content/image.jpg"}}