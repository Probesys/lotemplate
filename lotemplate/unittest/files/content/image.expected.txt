Test image modification
















Here we are after the image

Warning : this test does not test automatically the image insertion. You should check the result in the content folder in the image.unittest.odt to see if the image is well inserted.

It only test that the system can find the image on the disk.
